src/Symfony/Component/Form/Tests/FormViewTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Form\FormView;
+
+/**
+ * @author Barthold Bos <bartholdbos@hotmail.com>
+ */
+class FormViewTest extends TestCase
+{
+    /**
+     * @var FormView
+     */
+    private $formView;
+
+    protected function setUp()
+    {
+        $this->formView = new FormView();
```

Comment:
I don't think this is usefull: the instantiation should be done inline in each test case IMHO;